src/replay/consensus/cluster_sync.zig

```diff
@@ -584,7 +584,23 @@ fn processDuplicateConfirmedSlots(
             .duplicate_confirmed_hash = duplicate_confirmed_hash,
             .slot_status = status: {
                 if (progress.isDead(confirmed_slot) orelse false) break :status .dead;
-                const slot_hash = slot_tracker.get(confirmed_slot).?.state.hash.readCopy();
+                const slot_hash = if (slot_tracker.get(confirmed_slot)) |ref|
+                    ref.state.hash.readCopy()
+                else blk: {
+                    const highest_slot = if (slot_tracker.slots.count() > 0)
+                        std.mem.max(Slot, slot_tracker.slots.keys())
+                    else
+                        0;
+
+                    logger
+                        .debug()
+                        .logf(
+                        \\ duplicate confirmed slot not found in tracker: root={}, highest_tracked_slot={}, confirmed_slot={}
+                    ,
+                        .{ root, highest_slot, confirmed_slot },
+                    );
```

Comment:
I don't think we need to add extra computation, complexity, and coupling with the slot tracker, just to log a message about a very normal situation that will happen very often. I think `else null` is fine here.